docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -442,33 +442,15 @@ The matrix transform is also used to "correct" pin orderings into something that
 
 See the [in-tree keyboards](https://github.com/zmkfirmware/zmk/tree/main/app/boards/shields) that ZMK defines for examples of more complex matrix transformations.
 
-Also see the [matrix transform section](../../config/kscan.md#matrix-transform) in the Keyboard Scan configuration documentation for further details and examples of matrix transforms.
+Also see the [matrix transform section](../../config/layout.md#matrix-transform) in the Keyboard Scan configuration documentation for further details and examples of matrix transforms.
 
 ### Physical Layout
 
-The physical layout is the top level entity that aggregates all details about a certain possible layout:
+Your keyboard will need to have a physical layout defined.
```

Comment:
This isn't strictly a *need* and I am hesitant to push that for anyone just wanting to "get their keyboard working" and don't want/need alpha/beta level ZMK Studio at this point.

Reply:
I think we should discourage the use of a chosen `zmk,matrix-transform` in favor of physical transforms, and a bunch of the changes made were done with that in mind. That said, the *need* aspect should be for the basic physical transform node without the `keys` property - I'm thinking adding an "(Optional)" to the header describing the keys property.

Reply:
Yeah, I think that would work ok for me. Just don't want to unnecessarily complicate it for folks at this stage. Thanks!